abel/tools/symmetry.py

```diff
@@ -122,21 +119,17 @@ def get_image_quadrants(IM, reorient=True, symmetry_axis=None,
             vertical symmetry), you must reorient the image.')
 
     if 0 in symmetry_axis:   #  vertical axis image symmetry
-        Q0 = Q1 = (Q0*use_quadrants[0]+Q1*use_quadrants[1])/\
-                  (use_quadrants[0] + use_quadrants[1])
-        Q2 = Q3 = (Q2*use_quadrants[2]+Q3*use_quadrants[3])/\
-                  (use_quadrants[2] + use_quadrants[3])
+        Q0 = Q1 = Q0 + Q1
+        Q2 = Q3 = Q2 + Q3
 
-    if 1 in symmetry_axis:  # horizontal axis image symmetry
-        Q1 = Q2 = (Q1*use_quadrants[1]+Q2*use_quadrants[2])/\
-                  (use_quadrants[1] + use_quadrants[2])
-        Q0 = Q3 = (Q0*use_quadrants[0]+Q3*use_quadrants[3])/\
-                  (use_quadrants[0] + use_quadrants[3])
+    if 1 in symmetry_axis:   # horizontal axis image symmetry
+        Q1 = Q2 = Q1 + Q2
+        Q0 = Q3 = Q0 + Q3
 
     return Q0, Q1, Q2, Q3
 
 
-def put_image_quadrants(Q, odd_size=True, symmetry_axis=None):
+def put_image_quadrants(Q, original_image_shape, symmetry_axis=None):
```

Comment:
It makes sense that `put_image_quadrants` cannot, from the quadrants alone, figure out the size of the image. If it gets (3,3)-sized quadrants, it doesn't know if they came from a (5,5), (5,6), (6,5) or (6,6) image. I think that users will find it annoying that they need to specify the original image size, but there isn't an easy way around this. `abel.transform()` should wrap up the details nicely anyway, so perhaps it's not a big deal.